wpilibj/src/main/java/edu/wpi/first/wpilibj/GenericHID.java

```diff
@@ -41,10 +43,21 @@
 
     @SuppressWarnings("MemberName")
     public final int value;
+    private static final Map<Integer, HIDType> map = new HashMap<Integer, HIDType>();
```

Comment:
Don't need to specify the types in the constructor, the diamond operator lets the compiler infer the generic types:

```java
private static final Map<Integer, HIDType> map = new HashMap<>();
```